components/overflow-group/overflow-group.js

```diff
@@ -20,7 +20,18 @@ const OPENER_STYLE = {
 };
 
 function createMenuItem(node) {
-	const childText = node.text || node.firstChild && (node.firstChild.label || node.firstChild.text || node.firstChild.textContent.trim());
+	let childText = node.text;
+	if (!childText) {
+		const childNode = Array.from(node.childNodes).find(childNode => {
+			if (childNode.nodeType === Node.COMMENT_NODE) return false;
+			if (childNode.nodeType === Node.TEXT_NODE && !childNode.textContent.trim()) return false;
```

Comment:
If we don't care about this piece, I can rip it out and simplify this. The team that reported the bug was running into this, but the fix is to remove the space between `<d2l-button>` and `${this.localize...`.